gtsfm/runner/run_scene_optimizer_astronet.py

```diff
@@ -34,17 +37,46 @@ def construct_argparser(self) -> argparse.ArgumentParser:
         return parser
 
     def construct_loader(self) -> LoaderBase:
-        # Initialize loader.
+        """Initialize loader."""
         loader = AstronetLoader(
             data_dir=self.parsed_args.data_dir,
             use_gt_extrinsics=True,
             use_gt_sfmtracks=False,
             max_frame_lookahead=self.parsed_args.max_frame_lookahead,
             max_resolution=self.parsed_args.max_resolution,
+            gt_scene_mesh_path=self.parsed_args.scene_mesh_path,
         )
 
         return loader
 
+    def run(self) -> None:
+        """Run Structure-from-Motion (SfM) pipeline."""
+        # Create dask client.
+        cluster = LocalCluster(
+            n_workers=self.parsed_args.num_workers, threads_per_worker=self.parsed_args.threads_per_worker
+        )
+
+        with Client(cluster) as client, performance_report(filename="dask-report.html"):
+            # Scatter surface mesh across all nodes to preserve computation time and memory.
+            gt_scene_trimesh_future = client.scatter(self.loader.gt_scene_trimesh, broadcast=True)
```

Comment:
To be honest, the main reason I did it is because Dask told me to, but I can definitely say it saves a significant amount of computation time (although I don't have any exact numbers). 

The documentation is a little spotty, but from my understanding, it preallocates memory for the object on each worker and then passes around the pointer to the worker threads.

It definitely may be useful for other large objects in the pipeline.